composer.json

```diff
@@ -11,7 +11,8 @@
         "php-http/guzzle7-adapter": "^0.1.1",
         "promphp/prometheus_client_php": "^2.2.1",
         "grpc/grpc": "^1.30",
-        "google/protobuf": "^v3.3.0"
+        "google/protobuf": "^v3.3.0",
+        "nyholm/psr7": "^1.4"
```

Comment:
This is incorrect. The correct dependency here is [`psr/http-factory-implementation`](https://packagist.org/providers/psr/http-factory-implementation) (or it might be [`psr/http-message-implementation`](https://packagist.org/providers/psr/http-message-implementation) :thinking: )

You can add this package to `require-dev` to make the tests run.

Reply:
So I remember now

> https://github.com/guzzle/psr7/blob/master/src/HttpFactory.php

Hello @dkarlovi , thanks for this, it turns out that each time I run `composer require guzzlehttp/psr7` , it installs version `1.7-dev`

I will be getting something new so I can use the Factory